client/extensions/woocommerce/state/sites/coupons/handlers.js

```diff
@@ -51,10 +64,54 @@ export function requestCouponsSuccess( { dispatch }, action, { data } ) {
 	dispatch( couponsUpdated( siteId, params, body, totalPages, totalCoupons ) );
 }
 
+export function couponCreate( { dispatch }, action ) {
+	const { siteId, coupon } = action;
+	const path = 'coupons';
```

Comment:
Is this `const` needed since it is only used once below?

Reply:
It's basically just documentation. I didn't like having a stray 'coupons' string floating around in a parameter list.